resources/user.py

```diff
@@ -77,6 +81,12 @@ def patch(self,user_id):
         if not user:
             return {"Message": "Unable to find user."}, 400
       
+        if user_id != get_jwt_identity() and not get_jwt_claims()['is_admin']:
```

Comment:
I would like to see a couple tests added to verify this logic and prevent regression.
I currently see only one test that checks if an admin can update other users here: https://github.com/codeforpdx/dwellinglybackend/blob/development/tests/integration/test_users.py#L128

Can we add two more tests. One to check that a non-admin cannot change another users info. And another to check that a user can update their info and that they do receive an updated jwt upon a successful completion?

Reply:
@NickSchimek Sounds great. I'll write a couple of tests for the validations.